provision/provision.sh

```diff
@@ -274,6 +274,9 @@ cleanup_terminal_splash() {
   if [[ -f /etc/update-motd.d/50-landscape-sysinfo ]]; then
     rm /etc/update-motd.d/50-landscape-sysinfo
   fi
+  if [[ -f /etc/update-motd.d/80-live-patch ]]; then
+    rm /etc/update-motd.d/80-live-patch
```

Comment:
For my, without running this specifically i have the `80-livepatch` file. Maybe we should remove that aswell?

Reply:
Well this command will remove that file

Reply:
But the command here removes the file named 80-live-patch. I mentioned 80-livepatch.

Reply:
you are right... I didn't saw that. fixed.